Formula/xmrig.rb

```diff
@@ -0,0 +1,20 @@
+class Xmrig < Formula
+  desc ""
+  homepage "https://github.com/xmrig/xmrig"
+  url "https://github.com/xmrig/xmrig.git", :tag => "v2.2.1", :revision => "92d787c817dfa92ca38e36f634195f31ff7d40c3"
```

Comment:
We prefer building from source tarball (`https://github.com/xmrig/xmrig/archive/v2.2.1.tar.gz`) than git tag